x-pack/packages/kbn-cloud-security-posture-common/types/benchmark.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+export type BenchmarksCisId = 'cis_k8s' | 'cis_azure' | 'cis_aws' | 'cis_eks' | 'cis_gcp';
```

Comment:
let's either put it under `./types.ts` or another way around split `types.ts` into meaningful files, eg. `types/status.ts` and `types/finding.ts`